src/components/radio/RadioGroup.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+import RadioButton from './RadioButton';
+
+export interface RadioGroupProps {
+    children: Array<React.ReactElement> | React.ReactElement;
```

Comment:
React.ReactChild

Reply:
`ReactChild` causes a type error in `radio.props` on line 53, so we need `ReactElement` here. `ReactChild` can be of type `ReactElement` or `ReactText`, and `ReactText` does not have `props`.